### ISSUE TITLE
Reset encryption key

### ISSUE THREAD

**Issue by Eleonore9** [Laverna/laverna#371]
Is it possible to reset the encryption key?
What happens if we loose it?

**Comment by wwebfor** [comment]
No, it's not possible to reset encryption password without knowing the password.

**Comment by Eleonore9** [comment]
@wwebfor: ok, it makes sense! But I've cleaned my cache and I still can't get into Laverna in FF.

**Comment by wwebfor** [comment]
Did you clean your cache to remove everything and start over? If yes, then you need to remove IndexedDB database to start over:

![scrot](https://cloud.githubusercontent.com/assets/2404615/12686743/c063b9a6-c6f5-11e5-97f1-3b69af426a1e.png)

**Comment by Eleonore9** [comment]
:clap: thanks! It worked!
You might want to add that to the Readme for the next person having this question.

**Comment by mhmoodlan** [comment]
i had the same problem but i didn't know how to clean the cache or even indexedDB, can you help ?

**Comment by Eleonore9** [comment]
@mhmoodlan as it's written above after cleaning your cache you need to remove IndexedDB database.
In the console type:
```
indexedDB.deleteDatabase('notes-db')
```

**Comment by mhmoodlan** [comment]
i am using the windows app not the browser one, is that what i should do too in this case?.... i'm new to this electron thing so please be patient with me.

**Comment by wwebfor** [comment]
@mhmoodlan you can try to remove `%USERPROFILE%\AppData\Local\Chromium\Laverna` folder.

**Comment by hckweb** [comment]
@wwebfor okay so which folder for linux ?

**Comment by wwebfor** [comment]
@hckweb `~/.config/chromium/laverna`

**Comment by katwekibs** [comment]
@wwebfor  and for mac?

**Comment by t001** [comment]
@katwekibs I fixed my problem under mac with `rm -r ~/Library/Application\ Support/laverna/IndexedDB/`

**Comment by rosuH** [comment]
@wwebfor Thanks, I am using Windows10 x64, and I found that folder in `%USERPROFILE%\AppData\Roaming\Laverna` instead of `%USERPROFILE%\AppData\Local\Chromium\Laverna`.
 Hope this can be used as reference.
